frontend/src/api/node.tsx

```diff
@@ -4,14 +4,8 @@ import * as node from '../../types/api';
 
 export const getNode = async (): Promise<node.GetNodeResponse> => {
   try {
-    const accessToken = Cookies.get('access_token');
-    if (!accessToken) {
-      throw new Error('No access token found');
-    }
-
```

Comment:
this is just making it so you don't have to be logged in to see the node info right?